organization/models.py

```diff
@@ -62,6 +62,13 @@
 
 
 class OrganizationLinkToHashtag():
+    def __unicode__(self):
+        return "OrganizationLinkToHashtag"
+    
+    organization_we_vote_id = models.CharField(verbose_name="we vote permanent id", max_length=255, unique=True)
+    hashtag_text = models.CharField(verbose_name="hashtag text", max_length=255, unique=False)
```

Comment:
Maybe add a comment in the model specifying whether we are storing hashtags with the "#" or without?